### ISSUE TITLE
Pin version of requests package

### ISSUE THREAD

**Issue by stlozhaw** [sustainability-zhaw/sdg-indexer#7]
A bit of testing is required.

It looks like `gql[requests]` pulls a version of `requests` that is not compatible with `urllib 2.x`.

Result is the following error:
`cannot import name 'appengine' from 'requests.packages.urllib3.contrib'`

**Possible solutions:**
 - Pin the version of requests to something compatible with `gql 3.4.0` and `urllib 2.x`
 - Pin the version of `urllib` to `1.x`

**Comment by phish108** [comment]
Does the PR fix this issue?

**Comment by stlozhaw** [comment]
It does solve the problem but I have not looked if it is a known issue for gql. Only found information that the error is related to urllib3 and requests compatibility.

Looking at `gql` and `requests` requirements my guess is pip is not resolving the deps correctly and the following happens.

1. gql pulls `requests < 3`
2. `requests` pulls `urllib3 < 3` (In our case `urllib3 2.x` ignoring gql requirements for urllib3 < 2)?

**gql**
https://github.com/graphql-python/gql/blob/d140b830cd47fea5513b6bc30b1289c6416d25df/setup.py#LL44C1-L48C2

```python
install_requests_requires = [
    "requests>=2.26,<3",
    "requests_toolbelt>=0.9.1,<1",
    "urllib3>=1.26,<2",
]
```
**requests**
https://github.com/psf/requests/blob/6e5b15d542a4e85945fd72066bb6cecbc3a82191/setup.py#LL61C1-L66C2
```python
requires = [
    "charset_normalizer>=2,<4",
    "idna>=2.5,<4",
    "urllib3>=1.21.1,<3",
    "certifi>=2017.4.17",
]
```

**Comment by stlozhaw** [comment]
The setup.py snippet in the previous comment is from `>=3.4.1` with the fix. So pip is doing its job. 

Problem was fixed in `gql 3.4.1`. Upgrading it is the better fix.
https://github.com/graphql-python/gql/issues/412
